lib/lens/gzip_util.rb

```diff
@@ -0,0 +1,19 @@
+require 'zlib'
+require 'stringio'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.